jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -51,6 +61,36 @@ def get_server_era(pe_version) {
     }
 }
 
+def get_oss_server_era(oss_version) {
+    // TODO: eventually we will probably want to do something more sophisticated
+    //  here; currently only support 'latest'/'master'/'stable' OSS puppetserver,
+    //  and 'latest' agent
+    if (["latest", "master", "stable"].contains(oss_version)) {
+        return [type: "oss",
+                service_name: "puppetserver",
+                version: oss_version,
+                agent_version: "latest",
+                tk_auth: false,
```

Comment:
Yeah, that's true.  They do still ship with Ruby auth by default, not tk-auth.  Yeah, would also need to configure the `use-legacy-auth-conf` setting along with that - forgot that we weren't already doing that in this suite.  I'm good with revisiting that later if needed.